tests/test_imputation/test_impute_civil_defence.py

```diff
@@ -78,31 +73,213 @@ def create_input_df(self):
             "pg_sic_class",
             "empty_pgsic_group",
             "pg_class",
-            "empty_pg_group"]
+            "empty_pg_group",
+        ]
 
         data = [
-            [1001, 0, np.nan, np.nan, 0, "Clear", "800", "1234", "nan_1234", False, "nan", False],
-            [1001, 1, "C", "AC", 100, "Clear", "800", "1234", "AC_1234", False, "AC", False],
-            [1001, 2, "C", "AC", 200, "Clear", "800", "1234", "AC_1234", False, "AC", False],
-            [1001, 3, "D", "AC", 50, "Clear", "800", "1234", "AC_1234", False, "AC", False],
-            [1002, 0, np.nan, np.nan, 0, "Clear", "800", "1234", "nan_1234", False, "nan", False],
-            [1002, 1, "C", "AC", 100, "Clear", "800", "1234", "AC_1234", False, "AC", False],
-            [2002, 0, np.nan, np.nan, np.nan, "Clear", "800", "444", "nan_444", False, "nan", False],
-            [2002, 1, np.nan, "AC", 200, "Clear", "800", "444", "AC_444", True, "AC", False],
```

Comment:
Please could we reverse this to how it was before? The pre-commit has totally messed this up so it's not possible to read! Can we remove the line length check for tests?

Reply:
Good shout - I'll revert the commit and make sure it doesn't affect the tests.